cmd/solrays/util/util.go

```diff
@@ -0,0 +1,16 @@
+package util
```

Comment:
By convention, only the actual CLI package goes into cmd/ while any supporting libraries are in pkg/